### PR TITLE
[fabric] Support add-orderer-organization.yaml playbook

### LINKED ISSUE #2453: feat(fabric): Update adding a new orderer organization playbook to be compatible with version 2.5.4
Description
As a developer we want add orderer orgnazation playbook be compatible with fabric 2.5.4

Acceptance Criteria
Fabric 2.5.4 network should deploy with chaincode correctly.
Fabric 2.2.2 network should deploy with chaincode correctly.
Update documentation

### PR DESCRIPTION
**Pull request by mgCepeda**
**Primary Changes**

1. Fixed deployment add-orderer-organization.yaml playbook for fabric 2.2.2.
2. Added support for deploying add-orderer-organization.yaml playbook for Fabric 2.5.4.
3. Added deletion of genesis file in platforms/hyperledger-fabric/configuration/roles/delete/genesis role.

**Modifications**
platforms/hyperledger-fabric/configuration/add-orderer-organization.yaml 
platforms/hyperledger-fabric/configuration/cleanup.yaml 
platforms/hyperledger-fabric/configuration/deploy-network.yaml 
platforms/hyperledger-fabric/configuration/roles/create/configtx 
platforms/hyperledger-fabric/configuration/roles/create/genesis 
platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org 
platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org 
platforms/hyperledger-fabric/configuration/roles/create/orderers 
platforms/hyperledger-fabric/configuration/roles/delete/genesis 
platforms/hyperledger-fabric/configuration/roles/delete/vault_secrets

fixes #2453

### CODE REVIEW

File: platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org/appchannel/templates/update_channel_script.tpl [hunk 1]
```diff
@@ -3,9 +3,16 @@
 set -x
 
 CURRENT_DIR=${PWD}
+NETWORK_VERSION="{{ version }}"
 
-echo "installing jq "
-apt-get install -y jq
+if [ "$NETWORK_VERSION" != "2.5.4" ]; then
+    echo "installing jq "
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org/appchannel/templates/update_channel_script.tpl:9]: This should use the package_manager script

File: platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org/syschannel/templates/orderer_org.tpl [hunk 2]
```diff
@@ -5,7 +5,7 @@ set -x
 CURRENT_DIR=${PWD}
 
 echo "installing jq "
-apt-get install -y jq
+apk add jq
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org/syschannel/templates/orderer_org.tpl:8]: use package_manager

File: platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org/syschannel/templates/syschannel_update_address_script.tpl [hunk 3]
```diff
@@ -5,7 +5,7 @@ set -x
 CURRENT_DIR=${PWD}
 
 echo "installing jq "
-apt-get install -y jq
+apk add jq
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org/syschannel/templates/syschannel_update_address_script.tpl:8]: package manager

File: platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org/appchannel/templates/update_channel_script.tpl (hunk 1 above)
Comment (mgCepeda) [reply platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org/appchannel/templates/update_channel_script.tpl:9]: Done

File: platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org/syschannel/templates/orderer_org.tpl (hunk 2 above)
Comment (mgCepeda) [reply platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org/syschannel/templates/orderer_org.tpl]: Done

File: platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org/syschannel/templates/syschannel_update_address_script.tpl (hunk 3 above)
Comment (mgCepeda) [reply platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org/syschannel/templates/syschannel_update_address_script.tpl]: Done

**Review by sownak — APPROVED** [review]

**Pull request merged**